Assets/Scripts/Tank/TankSpawner.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TankSpawner : MonoBehaviour
```

Comment:
Only TankService needs access to this TankPositionService. So drag and drop this script in the Tank Service.